package.json

```diff
@@ -0,0 +1,56 @@
+{
+	"name": "backend",
+	"version": "1.0.0",
+	"description": "",
+	"scripts": {
+		"format": "npx prettier --write .",
+		"lint": "eslint --ext .js,.ts ./",
+		"build": "npx tsc",
+		"test": "jest --reporters=default --coverage --detectOpenHandles",
+		"ci": "npm run format && npm run lint && npm run build && npm run test",
+		"dev": "concurrently \"npx tsc -p tsconfig.build.json --watch\" \"nodemon dist/index.js\"",
+		"start": "node dist/index.js"
+	},
+	"keywords": [],
+	"contributors": [
+		"Jacob Turner",
+		"Brandon Kane",
+		"Lisa Li",
+		"Daisy Yang",
+		"Christovao Galambos",
```

Comment:
Name correction: Cristovao Galambos

Reply:
Fixed and in the pipeline. I'll wait to merge the changes before merging this pull request